### DISCUSSION TITLE
Add option to mute summary information

### DISCUSSION THREAD

**Discussion by jhsware** [vercel/vercel#3604] [Ideas]
When you use `now dns` in scripts there are summary lines that can't easily be redirected or muted. In some cases this makes it hard to create nice output.

Would be great if one could suppress these with an option.

```sh
$ now dns ls
> 18 Records found under username4 [2s]
[...]
```

**Comment by styfle** [comment]
Some of the output goes to stdout and some goes to stderr.

You can redirect stdout or stderr using file descriptor 1 and 2 respectively.

[See this example](https://www.cyberciti.biz/faq/redirecting-stderr-to-stdout/).

**Comment by jhsware** [comment]
You still get group headers, would be nice if you could choose a machine readable friendly version that just lists complete rows including domain name

```
domain.com         rec_efc67198bc6e066    ingress          A        93.226.155.122                      85d ago
domain.com         rec_efc67198bc6e066    ingress          A        93.226.155.122                      85d ago
```
So you can easily read it in a shell script with:

```
IFS=$'\n'
  for TMP in `now dns ls $DOMAIN` ; do
    IFS=' ' read DOMAIN ID NAME TYPE IPV4 TIME <<< $TMP
  done
```
Now you need to grep the input to filter out group headers.

**Comment by styfle** [comment]
You might want to try the API to retrieve this data: https://zeit.co/docs/api#endpoints/dns

**Comment by jhsware** [comment]
For shell scripts it is a lot easier to work with the CLI, JSON is a bit of a struggle. If I wrote a Node.js script I would certainly prefer the API.
